Marlin/src/lcd/dogm/status_screen_DOGM.cpp

```diff
@@ -366,6 +406,10 @@ void MarlinUI::draw_status_screen() {
       _draw_heater_status(-1, blink);
     #endif
 
+    #if HAS_HEATED_CHAMBER
+	    _draw_chamber_status(blink);
+	  #endif
```

Comment:
tabs fell in! ;-)

Reply:
![image](https://user-images.githubusercontent.com/145119/58665299-d6fbb600-8330-11e9-8c75-287d3efc9b1b.png)